examples/src/vue/examples/MultiStep.vue

```diff
@@ -0,0 +1,104 @@
+<script setup>
+import { FormKitSchema } from '@formkit/vue'
+
+const multiStepFormSchema = [
+{
+    $formkit: 'multi-step',
+    children: [
+      {
+        $formkit: 'step',
+        name: 'stepOne',
+        children: [
+          {
+            $formkit: 'text',
+            validation: 'required',
+          },
+        ],
+      },
+      {
+        $formkit: 'step',
+        name: 'stepTwo',
+      },
+      {
+        $formkit: 'step',
+        name: 'stepThree',
+      },
+    ],
+  },
+]
+</script>
+
+<template>
+  <FormKitSchema :schema="multiStepFormSchema" />
+
+  <FormKit
+    v-slot="{ value }"
+    type="form"
+  >
+    <FormKit type="step" />
+
+    <FormKit
+      type="multi-step"
+      tab-style="progress"
+      valid-step-icon="star"
+    >
+      <FormKit
+        type="step"
+        name="personalInfo"
+        valid-step-icon="bitcoin"
+      >
+        <FormKit
+          type="text"
+          label="My Name"
+          prefix-icon="avatarMan"
+          validation="required"
+        />
+        <FormKit
+          type="tel"
+          label="Phone"
+          prefix-icon="telephone"
+          validation="required"
+        />
+      </FormKit>
+    
+      <FormKit
+        type="step"
+        name="references"
+      >
+        <FormKit
+          type="textarea"
+          label="Please supply 2 references"
+          validation="required"
+        />
+      </FormKit>
+
+      <FormKit
+        type="step"
+        name="Supplemental"
+      >
+        <FormKit
+          type="textarea"
+          label="Why do you want to work here?"
+          validation="required"
+        />
+        <FormKit
+          type="radio"
+          label="How did you hear about us"
+          validation="required"
+          :options="[
+            { label: 'Google', value: 'google' },
+            { label: 'Facebook', value: 'facebook' },
+            { label: 'Twitter', value: 'twitter' },
+            { label: 'Friend', value: 'friend' },
+          ]"
+        />
+
+        <template #stepNext>
```

Comment:
This #stepNext, does it have something like `lastStep` boolean, or should we create another one for `#stepLast`, as imagine someone want to do all steps in-house, but want only the last step to submit.

For this case, it would make a submit call for each next step clicked right?

Reply:
Nope, steps actions are contained within each step, so your overrides are per-step.